sourcecode/apis/contentauthor/app/Http/Controllers/EmbedController.php

```diff
@@ -91,15 +92,19 @@ public function store(EmbedRequest $request)
             ]);
         }
 
-        $urlToCore = $this->getCoreBaseUrl($request->get('redirectToken'));
-
-        $responseValues = [
-            'url' => $urlToCore . "?" . http_build_query([
-                    "return_type" => "link",
-                    "url" => $inputs['link'],
-                ])
-        ];
-
-        return response()->json($responseValues, Response::HTTP_CREATED);
+        throw new Exception('remove me');
```

Comment:
Future task or forgot to remove?

Reply:
Future task. I've just created #2565 for it.